firmware/hw_layer/digital_input/trigger/trigger_input_adc.cpp

```diff
@@ -16,9 +16,7 @@
 #define DELTA_THRESHOLD_CNT_LOW (GPT_FREQ_FAST / GPT_PERIOD_FAST / 32)		// ~1/32 second?
 #define DELTA_THRESHOLD_CNT_HIGH (GPT_FREQ_FAST / GPT_PERIOD_FAST / 4)		// ~1/4 second?
 
-#if HAL_USE_ADC
```

Comment:
please do not remove ADC guard in firmware code?

Please rebase on master now that it has https://github.com/rusefi/rusefi/pull/5299/files - the idea is to be green without ADC.

this line here looks like it should be #if HAL_USE_ADC || EFI_UNIT_TEST unless unit test defines EFI_UNIT_TEST ?

overall when having unit tests considerations always safer to add " || EFI_UNIT_TEST " than to remove guards